back/spec/models/custom_field_spec.rb

```diff
@@ -354,9 +354,9 @@ def visit_sentiment_linear_scale(_field)
   describe 'description sanitizer' do
     it 'sanitizes script tags in the description' do
       custom_field = create(:custom_field, description_multiloc: {
-        'en' => '<p>Test</p><script>This should be removed!</script><p>But this should stay</p><a href="http://www.citizenlab.co" rel="nofollow">Click</a>'
+        'en' => '<p>Test</p><script>This content will stay</script><p>This content will also stay</p><a href="http://www.citizenlab.co" rel="nofollow">Click</a>'
       })
-      expect(custom_field.description_multiloc).to eq({ 'en' => '<p>Test</p>This should be removed!<p>But this should stay</p><a href="http://www.citizenlab.co" rel="nofollow">Click</a>' })
+      expect(custom_field.description_multiloc).to eq({ 'en' => '<p>Test</p>This content will stay<p>This content will also stay</p><a href="http://www.citizenlab.co" rel="nofollow">Click</a>' })
```

Comment:
Either we care about removing the content of script tags and we do it (which we currently do not), or we clearly document that we don't in the test. This change does the latter.